src/Compilers/CSharp/Test/Syntax/Parsing/RoleParsingTests.cs

```diff
@@ -795,115 +1444,187 @@ public void RoleParsing_WithParameterList_WithOneParameter(bool isExtension)
     }
 
     [Theory, CombinatorialData]
-    public void RoleParsing_WithoutBaseList(bool isExtension)
+    public void WithoutBody(bool isExplicit)
```

Comment:
Given that we've changed this for most other type declarations in the primary-ctor branch, we probably want to allow the semi-colon body here too. Worth leaving a prototype to followup after primary-ctors are merged.

Reply:
There's already a PROTOTYPE comment below